examples/example.krun

```diff
@@ -73,3 +73,7 @@ TEMP_READ_PAUSE = 1
 
 # CPU pinning (off by default)
 #ENABLE_PINNING = False
+
+AMPERF_RATIO_BOUNDS = 0.97, 1.03
+AMPERF_BUSY_THRESHOLD = 3.4 * 1024 * 1024 * 1024  # 3.6 GHz
```

Comment:
On our machines APERF happens to increment at the CPU base frequency (but we cannot rely on this in general)